Bartender drinks content

ZeroDayDaemon opened the pull request:
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->
Adds moonshine drinks sprites
Sprites courtesy of @Mith-randalf.

And now addressing the elephant in the room with this PR:

Lean and Leanshine.

Lean was removed due to its ability to poison and other wacky effects, as well as it being "annoying".
I feel that its complete removal was entirely not needed, and a rework or removal of the ability to craft it would have had the exact same intended outcome.
so that's exactly what I've done, I removed the effects, and make base lean uncraftable for the time being, until the time is right when it can be re-added as a drink that can be crafted.

Having healthy discussion is welcome on this PR, what is not welcome is closing it without reasonable community and maintainer discussion.

I have Re-Added Lean and Leanshine and given them sprites.


When #6961 is completed I also plan to PR 3 bottles of moonshine and 1 leanshine as emag inventory items for the bartenders drink bottle machine

**Screenshots**
<!-- If applicable, add screenshots to showcase your PR. If your PR is a visual change, add
screenshots or it's liable to be closed by maintainers. -->

**Changelog**
<!--
Here you can fill out a changelog that will automatically be added to the game when your PR is merged
There are 4 icons for changelog entries: add, remove, tweak, fix. I trust you can figure out the rest.

You can put your name after the :cl: symbol to change the name that shows in the changelog (otherwise it takes your GitHub username)
Like so: :cl: PJB

Generally, only put things in changelogs that players actually care about. Stuff like "Refactored X system, no changes should be visible" shouldn't be on a changelog.

For writing actual entries, don't consider the entry type suffix (e.g. add) to be "part" of the sentence:
bad: - add: a new tool for engineers
good: - add: added a new tool for engineers
-->

:cl: Daemon
- add: Added Moonshine drink sprites
- add: Re-Added fun!

moonheart08 commented:
https://github.com/tgstation/tgstation/pull/65464

y'know i think i agree with /tg/ for once

moonheart08 commented:
For me to really be OK with lean being readded it needs renamed and recolored as to not be the meme of the month.

Snowni commented:
I'd rather keep lean out, shit meme, and it's borderline RDM when the bartenders stock their shelves with it and people drink it. Not to mention all the people forcefeeding it to others.

ZeroDayDaemon commented:
> For me to really be OK with lean being readded it needs renamed and recolored as to not be the meme of the month, so we don't get people going crazy over haha purple

It can't be crafted. This is no longer an issue. Also renaming it makes no sense, since lean is an actual real world thing.

ZeroDayDaemon commented:
> I'd rather keep lean out, shit meme, and it's borderline RDM when the bartenders stock their shelves with it and people drink it. Not to mention all the people forcefeeding it to others.

It was in the game since 2020 with no issue, also, it has no effects in this PR, so, yeah.

Leander-0 commented:
> I'd rather keep lean out, shit meme, and it's borderline RDM when the bartenders stock their shelves with it and people drink it. Not to mention all the people forcefeeding it to others.

if you want to remove lean just because its a meme then you should remove Skub too.

i really dont see why all the hate to lean all of a sudden.

ZeroDayDaemon commented:
> https://github.com/tgstation/tgstation/pull/65464
> 
> y'know i think i agree with /tg/ for once

For once I agree with swept on that PR.

moonheart08 commented:
> > I'd rather keep lean out, shit meme, and it's borderline RDM when the bartenders stock their shelves with it and people drink it. Not to mention all the people forcefeeding it to others.
> 
> if you want to remove lean just because its a meme then you should remove Skub too.
> 
> i really dont see why all the hate to lean all of a sudden.

Because it just suddenly blew up and caused our game staff a **fuckton** of unnecessary work. They otherwise have no way of combatting it as it's an LRP server, so even if half the crew is dead of severe drug poisoning from consuming lean repeatedly they just have to live with the load. The easiest/sanest route to handling something being super painful to work with like this is just to remove it. 

Also, if it has _no_ effects, then it's just not eligible to be in the game, as it's an unattainable, useless chem.

ZeroDayDaemon commented:
Lean has been in the game for almost two years. And only now do people have an issue with it, because I will admit I fucked up by making it a poison. But, It is not a poison in this PR, nor can it be crafted.

Mith-randalf commented:
idgaf about the lean but if this is unmerged 24 hours from now I'll resprite the moonshine jug, looking at it, it's too small (but if it does get merged its fine cos it really doesnt matter)

ZeroDayDaemon commented:
So I believe I've addressed every issue and concern with it in this PR, except one:

"I don't like it, it's an unfunny meme"
That's your opinion, if we removed everything from the game that one person didn't find funny or fun, there would be no game.

Leander-0 commented:
> Because it just suddenly blew up and caused our game staff a **fuckton** of unnecessary work. They otherwise have no way of combatting it as it's an LRP server, so even if half the crew is dead of severe drug poisoning from consuming lean repeatedly they just have to live with the load. The easiest/sanest route to handling something being super painful to work with like this is just to remove it.
> 
> Also, if it has _no_ effects, then it's just not eligible to be in the game, as it's an unattainable, useless chem and we no longer allow that in the codebase.

![drugs](https://user-images.githubusercontent.com/80334192/158495681-6887fa2d-192c-4053-a12d-57ec469bc230.png)

When a player consumes drugs be it Meth, a blunt or lean they all know the effects and the risks of taking of taking such drugs.

THC and and smoking have no effects right now and yet the players smoke cigars and make blunts.

what will happen when other drugs like cocaine get added into the game then?

ZeroDayDaemon commented:
I've reworked your concerns with lean, It now has a use and can be crafted, has been moved to soda since thats the primary filler ingredient, and cures space cold

ZeroDayDaemon commented:
crafting it is more difficult and now requires access to a chemistry dispenser

Fishfish458 commented:
Guess I'll throw in my comment here too, but I don't think the "It has no purpose/ is dangerous to the crew" argument really has grounding in a game that adds massive varieties of chemicals/foods/drinks/items purely for additional content and RP flavor.  in SS13 are drinks that can poison or burn you, foods that immediately kill your character or turn them into a borg, chemicals that can make you mind slaved to someone else.

I agree that the Lean meme is pretty stupid and if references to drugs truly need to be toned down, sure, rename it or whatever, though there are tons more dangerous food/drinks/chemicals that still should be added, not that they might be unique to their interactions but because they offer variety and something to roleplay with.

metalgearsloth commented on Resources/Prototypes/Reagents/medicine.yml:
```diff
@@ -77,6 +77,21 @@
           groups:
             Brute: -2
 
+- type: reagent
+  id: Codeine
+  name: codeine
```
Do we have any other references to actual drugs? If so they should be culled.

ZeroDayDaemon replied:
Meth, Marijuana, Alcohol, Space Drugs, uhhhh, alot more i think

ZeroDayDaemon replied:
Nicotine

ZeroDayDaemon replied:
Codeine is usually the main active ingredient in prescription cough medicine

metalgearsloth replied:
All of those are restricted to varying degrees so it's a bit weird.

metalgearsloth reviewed (commented):
Putting in a real world poison, lean, seems like a bad idea.

ZeroDayDaemon commented:
> Putting in a real world poison, lean, seems like a bad idea.

What on your thoughts on why that is?

moonheart08 commented:
> Guess I'll throw in my comment here too, but I don't think the "It has no purpose/ is dangerous to the crew" argument really has grounding in a game that adds massive varieties of chemicals/foods/drinks/items purely for additional content and RP flavor. in SS13 are drinks that can poison or burn you, foods that immediately kill your character or turn them into a borg, chemicals that can make you mind slaved to someone else.
> 
> I agree that the Lean meme is pretty stupid and if references to drugs truly need to be toned down, sure, rename it or whatever, though there are tons more dangerous food/drinks/chemicals that still should be added, not that they might be unique to their interactions but because they offer variety and something to roleplay with.

References to drugs are fine

the problem is lean is a massive meme right now. I can demonstrate why this is a problem at any time if needed (by temporarily readding it to the game using admin tools. Half the crew will be dead in 10 minutes.)

ZeroDayDaemon commented:
> > Guess I'll throw in my comment here too, but I don't think the "It has no purpose/ is dangerous to the crew" argument really has grounding in a game that adds massive varieties of chemicals/foods/drinks/items purely for additional content and RP flavor. in SS13 are drinks that can poison or burn you, foods that immediately kill your character or turn them into a borg, chemicals that can make you mind slaved to someone else.
> > I agree that the Lean meme is pretty stupid and if references to drugs truly need to be toned down, sure, rename it or whatever, though there are tons more dangerous food/drinks/chemicals that still should be added, not that they might be unique to their interactions but because they offer variety and something to roleplay with.
> 
> References to drugs are fine
> 
> the problem is lean is a massive meme right now. I can demonstrate why this is a problem at any time if needed (by temporarily readding it to the game using admin tools. Half the crew will be dead in 10 minutes.)

I've basically given up on lean at this point, im more confused about sloths aversion to codeine, when theres already a load of dangerous drugs in game, real ones, like alcohol and meth and nicotine

ZeroDayDaemon commented:
> > > Guess I'll throw in my comment here too, but I don't think the "It has no purpose/ is dangerous to the crew" argument really has grounding in a game that adds massive varieties of chemicals/foods/drinks/items purely for additional content and RP flavor. in SS13 are drinks that can poison or burn you, foods that immediately kill your character or turn them into a borg, chemicals that can make you mind slaved to someone else.
> > > I agree that the Lean meme is pretty stupid and if references to drugs truly need to be toned down, sure, rename it or whatever, though there are tons more dangerous food/drinks/chemicals that still should be added, not that they might be unique to their interactions but because they offer variety and something to roleplay with.
> > 
> > 
> > References to drugs are fine
> > the problem is lean is a massive meme right now. I can demonstrate why this is a problem at any time if needed (by temporarily readding it to the game using admin tools. Half the crew will be dead in 10 minutes.)
> 
> I've basically given up on lean at this point, im more confused about sloths aversion to codeine, when theres already a load of dangerous drugs in game, real ones, like alcohol and meth and nicotine

also, only half the crew will be dead if lean is a poison again, which, in this PR, it is not

ZeroDayDaemon commented:
I'm going to be expanding and reworking this PR since i have the time today

ZeroDayDaemon commented:
Closing this, maybe another time
